Makefile

```diff
@@ -50,4 +50,44 @@ build-anp:
 	mv $(PERMANENT_TMP)/$(ANP_NAME)/proxy-server ./
 .PHONY: build-anp
 
-# TODO include ./test/integration-test.mk
+# e2e
+
+HELM?=_output/linux-amd64/helm
+KUBECTL?=kubectl
+
+IMAGE=quay.io/open-cluster-management/cluster-proxy-addon:latest
+IMAGE_PULL_POLICY=Always
+CLUSTER_BASE_DOMAIN=
+
+ensure-helm:
```

Comment:
cluter-proxy-addon-chart is depend on openshift (etc route), I don't know whether/how can we install a openshift on prow.

Reply:
prow use ocp by default

Reply:
yes, another concern is the main purpose of e2e is for developer to do local tests, if we can make some modifications to suite [kind cluster + all-in-one-ocm] environment, it would be convenience for local-test.

Reply:
if you want to make the addon support all env, I think we need enhance our code/chart, rather than using e2e to support, right?

and the e2e ensure our code work, so I think it should include all components

Reply:
Yes, that's right, I will modify chart repo to support local-test, and using default(ocp) mode for prow e2e.

Reply:
you may try the chart in none-ocp cluster, I think it should be able to deploy the components except the ocp routes, then we also can do some test with it, right?

Reply:
Sure, we could do that as well.